app/views/content/train-to-be-a-teacher/_why-teach.html.erb

```diff
@@ -1,11 +1,9 @@
-<section class="why-teach" aria-label="Why teach?">
+<section class="why-teach" aria-label="Inspire the next generation">
```

Comment:
@garethwilliamsdesign This component doesn't appear on the front end by the looks of it - should we just delete this file altogether?

Reply:
Sure. I'll do that after UCD.

Reply:
Isn't `app/views/content/how-to-apply-for-teacher-training/_why-teach.html.erb` the one on the How to apply page?